src/style.css

```
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: normal;
+  font-size: 600;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
```

```suggestion
html {
  font-family: "Avenir", sans-serif;
}
```
always set the font to the `html` tag and inherit where needed